pkgs/by-name/ca/cabal-cli/package.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
+
+buildNpmPackage rec {
+  pname = "cabal-cli";
+  version = "15.0.2";
```

Comment:
Consider using `let .. in` instead of `rec`, e.g.

```nix
let
  version = "15.0.2";
in 
buildNpmPackage {
  inherit version;
  ...
}

Reply:
Thanks, I have fixed it